src/score_table_models.py

```diff
@@ -129,22 +129,39 @@ class Region(Base):
     created_at = Column(DateTime, nullable=False)
     updated_at = Column(DateTime)
 
+    metrics = relationship('ExperimentMetric', back_populates='region')
+
 
 class MetricType(Base):
     __tablename__ = METRIC_TYPES_TABLE
+    __table_args__ = (
+        UniqueConstraint(
+            'name',
+            'measurement_type',
+            'measurement_units',
+            'stat_type',
+            name='unique_metric_type'
+        ),
+    )
     
     id = Column(Integer, primary_key=True, autoincrement=True)
     name = Column(String(128), nullable=False)
-    measurement_type = Column(String(32), nullable=False)
-    measurement_units = Column(String(32))
-    stat_type = Column(String(32))
-
-    # metrics = relationship('ExperimentMetric', backref='metric_type')
+    measurement_type = Column(String(64), nullable=False)
+    measurement_units = Column(String(64))
+    stat_type = Column(String(64))
+    description = Column(JSONB(astext_type=sa.Text()), nullable=True)
+    created_at = Column(DateTime, nullable=False)
+    updated_at = Column(DateTime)
+    
```

Comment:
Also need to add another column that will document the averaging period. E.g. for atm. it is 6 hours. For ocean it is 24 hours.

Reply:
We discussed this and came to the determination to assess this after using the tool for a trial period.